### DISCUSSION TITLE
Scaling/Device Independent units

### DISCUSSION THREAD

**Discussion by lylerolleman** [AvaloniaUI/Avalonia#12858] [Q&A]
I was wondering if there were any nice ways of handling global UI scaling per platform or another cleaner means of making the same UI look reasonable on other devices.

If for example I create a new Avalonia app with the default template and run it, it looks fine on desktop, but is miniscule on a mobile device. 

I know there is the OnPlatform markup extension but I'd rather not use it favour of something more global/sensible.

**Comment by timunie** [comment]
I guess Avalonia.Behaviors may have something for you. Also check for example how ControlCatalog does change the look and feel of the hamburger menu during scaling: 
https://github.com/AvaloniaUI/Avalonia/tree/master/samples/SampleControls/HamburgerMenu
